bundles/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/handler/SerialBridgeConfiguration.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.serial.internal.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.io.transport.serial.SerialPort;
+
+/**
+ * Class describing the serial bridge user configuration
+ *
+ * @author Mike Major - Initial contribution
+ */
+@NonNullByDefault
+public class SerialBridgeConfiguration {
+    /**
+     * Serial port name
+     */
+    public @Nullable String serialPort;
+
+    /**
+     * Serial port baud rate
+     */
+    public int baudRate = 9600;
+
+    /**
+     * Serial port data bits
+     */
+    public int dataBits = 8;
+
+    /**
+     * Serial port parity
+     */
+    public String parity = "N";
```

Comment:
Are you suggesting changing parity to be an enum rather than a string?

How would you handle a user setting an invalid value in the config. I've tried it and you get a an IllegalArgumentException thrown from ConfigMapper.

Reply:
But that's up to the user if he/she configures wrong values. Your code is much simpler and less error prone if you use the enum.